server/metrics.go

```diff
@@ -36,9 +36,12 @@ var (
 		"/com.docker.api.protos.containers.v1.Containers/Kill":    "kill",
 		"/com.docker.api.protos.containers.v1.Containers/Inspect": "inspect",
 		"/com.docker.api.protos.containers.v1.Containers/Logs":    "logs",
-		"/com.docker.api.protos.streams.v1.Streaming/NewStream":   "",
+		"/com.docker.api.protos.streams.v1.Streaming/NewStream":   "streaming",
```

Comment:
Why this change?

Reply:
I just didn't want to let this empty, if anyone calls this, we'll see something we can follow in the metrics